packages/api-explorer/src/components/common/common.tsx

```diff
@@ -30,9 +30,8 @@ import { Heading } from '@looker/components'
  * Common styled components to be used across the whole library
  */
 
-export const ApixHeading = styled(Heading)`
-  color: ${({ theme }) => theme.colors.text5};
```

Comment:
This is the default color already :)

Reply:
@lukelooker Good to know, this was here because we were using a different color previously that ended up lighter at one point during the updates so then this was changed to account for that. I assume this could just be removed entirely now?

Reply:
The default (below) are still in play (setting `fontWeight: bold` and some presets for margin and padding so the component is still doing some work. Still pondering how this type of use case could be handled long-term but for now those defaults continue to make this relevant.

Reply:
@lukelooker sorry, I should have been more specific, I meant only removing the color value.